bin/wskadmin

```diff
@@ -1 +0,0 @@
-../tools/admin/wskadmin
```

Comment:
Please remove this change. You can use `git stash` to prevent adding changes that we don't want to be added to a commit.